### ISSUE TITLE
Please clarify license

### ISSUE THREAD

**Issue by lbmn** [yglukhov/nimx#158]
TL;DR: please add the [BSD license file](http://copyfree.org/content/standard/licenses/2bsd/license.txt).  

There is no license file for this project.  The nimble metadata says "BSD", which would have been sufficient for a license that doesn't have [vastly different variants](https://en.wikipedia.org/wiki/BSD_licenses#Terms), but there's ambiguity if you mean BSD2, BSD3, or the horribly restrictive BSD4.  I personally use the [Unlicense](https://en.wikipedia.org/wiki/Public-domain_software#Public-domain-like_licenses_and_waivers) for all my works, but any license on [the copyfree.org white-list](http://copyfree.org/standard/licenses) would also be OK.

I am promoting Nim as, among its many other virtues, a programming language with the most no-hassle unencumbered no-lawyers-needed permissively-licensed ecosystem.  89% of nimble packages fit the copyfree.org definition of free software, compared for example to 81% to runner-up cabal or 64% for npm.  Towards this end, it is important to resolve any ambiguities in the licensing. 

Thank you very much for all your great contributions to the Nim ecosystem!  :+1:

**Comment by lbmn** [comment]
Thank you very much.  :smile:
